content/using-atom/sections/github-package.md

```diff
@@ -85,25 +92,29 @@ Once you've staged your changes, enter a commit **message**. Feel free to descri
 If you need more space, click the expand icon at the bottom right. It will open a commit editor in the center.
 
 
```

Comment:
is it worth calling out the co author functionality here? (Totally ok if that's too granular to mention)

Reply:
👍 Yessss..

I added the following:

> To add multiple **co-authors** to a commit, click the "👤➕" icon in the bottom left corner of the commit message editor. Then search by name, email or GitHub username to give credit to a co-author.

![github-commit-with-co-authors](https://user-images.githubusercontent.com/378023/45079556-d29dc600-b12d-11e8-9248-00ca1effbd36.png)